### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KamranAsif**
## Description
Added Travis CI status, coverall test coverage and npm version badges 

## Motivation and Context
Requested here: https://github.com/schrodinger/fixed-data-table-2/pull/29

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by KamranAsif** [comment]
Removing coveralls support, since that requires extra config within our testing system

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Fixed Data Table 2 for React
+Fixed Data Table 2 for React ![Travic CI status](https://api.travis-ci.org/schrodinger/fixed-data-table-2.svg) [![npm version](https://badge.fury.io/js/fixed-data-table-2.svg)](https://www.npmjs.com/package/fixed-data-table-2)
 ====================================
 
-### Fixed-Data-Table-2 is a continuation of [facebook/fixed-data-table](https://github.com/facebook/fixed-data-table).  The original repo is no longer maintained and has many pull requests awaiting response.
+#### Fixed-Data-Table-2 is a continuation of [facebook/fixed-data-table](https://github.com/facebook/fixed-data-table).  The original repo is no longer maintained and has many pull requests awaiting response.
```
Comment (wcjordan) [inline README.md]: is this intended? whats it do?

**Comment by wcjordan** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (KamranAsif) [inline README.md]: Just make it smaller text

**Pull request merged**